modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPkgPrimaryKeyFinderTestSetup.java

```diff
@@ -32,11 +32,21 @@ protected void createMetadataTable() throws Exception {
                         + "pk_column_idx int, pk_policy varchar(255), pk_sequence varchar(255))");
     }
 
+    @Override
+    protected void createMetadataView(String name) throws Exception {
```

Comment:
The primary key metadata is a cross database behavior, should be tested in all of them. If you remove the case sensitiveness business it should be actually easy to add the code for other databases.

Reply:
Done.